chain/tests/process_block_cut_through.rs

```diff
@@ -56,8 +56,10 @@ where
 		chain.set_prev_root_only(&mut block.header)?;
 
 		// Manually set the mmr sizes for a "valid" block (increment prev output and kernel counts).
-		block.header.output_mmr_size = pmmr::insertion_to_pmmr_index(prev.output_mmr_count() + 1);
-		block.header.kernel_mmr_size = pmmr::insertion_to_pmmr_index(prev.kernel_mmr_count() + 1);
+		// I don't see how 1 + 0-based index corresponds to a completed size
+		// E.g. output_mmr_count==3 gives size 5
+		block.header.output_mmr_size = 1 + pmmr::insertion_to_pmmr_index(prev.output_mmr_count());
+		block.header.kernel_mmr_size = 1 + pmmr::insertion_to_pmmr_index(prev.kernel_mmr_count());
```

Comment:
size *is* position, but it's the position of the next node to be added, rather than the last node added.
this has the advantage that it also works for an empty MMR, where the old rule had an exception.